data-raw/make-weights.R

```diff
@@ -44,6 +44,75 @@ for (w in names(.maps)) {
       metacanc = 6,
       aids = 4
     )
+  } else if (grepl(pattern = "m3", x = w)) {
+    # m3 weights
+    .weights[[w]][["m3"]] <- c(
+      ami = 0.197491908,
+      chf = 0.539809861,
+      pvd = 0.349250005,
+      aneur = 0.260195993,
+      ven = 0.214050369,
+      cevd = 0.097803808,
+      dementia = 1.021975368,
+      bd = 0.039711074,
+      copd = 0.6253395,
+      conntiss = 0.290446442,
+      pud = 0.152986438,
+      diabunc = 0.299383867,
+      diabc = 0.271607393,
+      para = 0.281895685,
+      rend = 0.334155906,
+      canccolrec = 0.372878764,
+      cancbreast = 0.411891435,
+      cancprost = 0.432343447,
+      canclung = 1.972481401,
+      canclymphleuk = 1.190108503,
+      cancuppergi = 1.941498638,
+      cancmela = 0.342233292,
+      cancgyn = 0.70658858,
+      cancoth = 1.103452294,
+      metacanc = 2.468586878,
+      msld = 0.474321939,
+      aids = 0.452647425,
+      ang = 0, #-0.082399267 
+      hypunc = 0.117746303,
+      carit = 0.173859876,
+      pcd = 0.398432833,
+      valv = 0.256577208,
+      bdi = 0.086960591,
+      ond = 0.564391512,
+      epi = 0.594991823,
+      mpnd = 0.208276284,
+      mpd = 0.212789563,
+      anxbd = 0.121481351,
+      blood = 0.265142145,
+      dane = 0.180927466,
+      obes = 0.248243722,
+      alcohol = 0.576907507,
+      drug = 0.558979499,
+      panc = 0, #-0.237983891
+      endo = 0.112673001,
+      utc = 0.046548658,
+      tub = 0,  #-0.104290289
+      bone = 0.132827597,
+      osteounc = 0.083506878,
+      immsys = 0.398529751,
+      metab = 0.006265195,
+      mentret = 1.405761403,
+      cvhep = 0.569092852,
+      sleep = 0.245749995,
+      inear = 0.06090681,
+      cinfnos = 0,  #-0.237983891
+      maln = 0.331335106,
+      ceye = 0.179923774,
+      cdnos = 0, #-0.104225698
+      intest = 0, #-0.254089697
+      jsd = 0.095585857,
+      # Zero weight exclusion/complication flags
+      flag_comp_diab = 0, # Diabetes
+      flag_exc_osteo = 0, # Osteoporosis
+      flag_exc_hyp = 0 # Hypertension
```

Comment:
Same as above, could you please re-arrange these to be in the same order of Supplementary Table 5 in the Stanley and Sarfati paper?

Reply:
Good suggestion, the latest commit has this implemented.

Reply:
Super, thanks!